README.md

```diff
@@ -119,6 +119,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 * [Bamboo](https://www.atlassian.com/software/bamboo) - Atlassian's solution with good integration of other products. You can either apply for an Open Source license or buy it.
 * [CircleCI](https://circleci.com/) - Hosted service with a free trial.
 * [Codeship](https://www.codeship.io/features) - Hosted services with a limited free plan.
+* [DotCi](https://github.com/groupon/DotCi) - It provides GitHub build management and continuous integration through docker on Jenkins.
```

Comment:
Please read the discussion in the issues. I don't know if we should add more commercial cloud CIs. They have practically no adaption.